.github/ISSUE_TEMPLATE/performance.md

```diff
@@ -48,6 +48,18 @@ assignees: ''
        https://medium.com/flutter-io/profiling-flutter-applications-using-the-timeline-a1a434964af03#a499
 -->
 
+<!--
+     Please tell us on which host platform you use to build / debug your app
+     Which target platform(s) the problem occurs (Android / iOS / Web / macOS / Linux / Windows)
+     Which target OS version, for Web, browser, is the test system running?
+     Does the problem occur on emulator/simulator as well as on physical devices?
+-->
+
+**Host Platform:**
```

Comment:
Why are we asking for host platform when we ask for `flutter doctor -v`?

Reply:
I just followed your recommendation to make it clearer what's host, what's target

Reply:
But I can remove that part again

Reply:
My recommendation was to make it clearer that you were asking about the target so that they *don't* give us the host.